src/Processors/Formats/Impl/NativeORCBlockInputFormat.cpp

```diff
@@ -312,6 +797,9 @@ Chunk NativeORCBlockInputFormat::generate()
     if (is_stopped)
         return {};
 
+    /// TODO: figure out why reuse batch would cause asan fatals in https://s3.amazonaws.com/clickhouse-test-reports/55330/be39d23af2d7e27f5ec7f168947cf75aeaabf674/stateless_tests__asan__[4_4].html
+    /// Not sure if it is a false positive case. Notice that reusing batch will speed up reading ORC by 1.15x.
```

Comment:
I don't understand why asan fatals appears if `batch` is reused. Maybe it is a false positive ? @al13n321